package.json

```diff
@@ -21,8 +21,8 @@
         "url": "https://github.com/microsoft/powerquery-formatter/issues"
     },
     "description": "A source code formatter for the Power Query/M formula language.",
-    "main": "lib/index.js",
-    "types": "lib/index.d.ts",
+    "main": "lib/powerquery-formatter/index.js",
+    "types": "lib/powerquery-formatter/index.d.ts",
```

Comment:
Oops. I wonder if there is an easy way to add an automated test for this. Must be a common problem.

Reply:
There probably is. Invoke a random .js file which only imports the root. I would say it's not worth investigating right now since we're not going to be doing a restructuring any time soon.